src/views/Search.vue

```diff
@@ -25,9 +24,9 @@
             <div class="rf-grid-row">
                 <form @submit.stop.prevent="Search(newResearch)" class="searchBar rf-col">
                     <div class="rf-search-bar rf-search-bar--lg" id="search-input--lg">
-                      <label class="rf-label" for="search-input--lg-input">Label de la barre de recherche</label>
-                      <input type="search" class="rf-input" id="search-input-input" name="search-input-input" v-model="newResearch" placeholder="Rechercher">
-                      <button class="rf-btn" title="Rechercher" type="submit">
+                      <label class="rf-label" for="search-input--lg-input">Rechercher un financement</label>
+                      <input type="search" class="rf-input" id="search-input-input" name="search-input-input" v-model="newResearch" placeholder="Rechercher un mot, une expression, une référence...">
```

Comment:
Utilisation du caractère typographique « … » au lieu de trois « . ».

```suggestion
                      <input type="search" class="rf-input" id="search-input-input" name="search-input-input" v-model="newResearch" placeholder="Rechercher un mot, une expression, une référence…">
```

Reply:
Bien vu